### PR TITLE
facet python doc

### PR DESCRIPTION
**Pull request by Kully**

### CODE REVIEW

**Comment by Kully** [comment]
@cldougl shall I wait to merge https://github.com/plotly/plotly.py/pull/776 before merging this?

File: _posts/python/statistical/facet-trellis/facet-and-trellis-plots.ipynb [hunk 1]
```diff
@@ -0,0 +1,762 @@
...
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "mpg = pd.read_table('https://raw.githubusercontent.com/plotly/datasets/master/mpg_2017.txt')\n",
```
Comment (cldougl) [inline _posts/python/statistical/facet-trellis/facet-and-trellis-plots.ipynb]: I would put the dataset with the specific example that it's with so users don't have to import more than necessary / can cut down on copy & pasting

File: _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+    <span class="n">facet_col</span><span class="o">=</span><span class="s1">&#39;smoker&#39;</span><span class="p">,</span>
+<span class="p">)</span>
+
+<span class="k">for</span> <span class="n">trace</span> <span class="ow">in</span> <span class="n">fig</span><span class="p">[</span><span class="s1">&#39;data&#39;</span><span class="p">]:</span>
+    <span class="n">trace</span><span class="p">[</span><span class="s1">&#39;type&#39;</span><span class="p">]</span> <span class="o">=</span> <span class="s1">&#39;histogram&#39;</span>
+    <span class="k">del</span> <span class="n">trace</span><span class="p">[</span><span class="s1">&#39;marker&#39;</span><span class="p">][</span><span class="s1">&#39;size&#39;</span><span class="p">]</span>
+    <span class="k">del</span> <span class="n">trace</span><span class="p">[</span><span class="s1">&#39;mode&#39;</span><span class="p">]</span>
+
+<span class="n">py</span><span class="o">.</span><span class="n">iplot</span><span class="p">(</span><span class="n">fig</span><span class="p">,</span> <span class="n">filename</span><span class="o">=</span><span class="s1">&#39;facet - bar traces&#39;</span><span class="p">)</span>
```
Comment (cldougl) [inline _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html]: I wouldn't call this bar traces cause it's a histogram and we don't want to confuse people that they can plot bar graphs if only histograms are available

File: _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+    import pandas as pd
+    
+    mtcars = pd.read_csv(&#39;https://raw.githubusercontent.com/plotly/datasets/master/mtcars.csv&#39;)
+    
+    fig = ff.create_facet_grid(
+        mtcars,
+        x=&#39;wt&#39;,
+        y=&#39;mpg&#39;,
+        facet_col=&#39;cyl&#39;,
+        facet_col_labels={4: &#34;$\alpha$&#34;, 6: &#39;$\beta$&#39;, 8: &#39;$\sqrt[y]{x}$&#39;},
```
Comment (cldougl) [inline _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html]: I would make these labels coincide/relate to the graph in the example

File: _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html [hunk 4]
```diff
@@ -0,0 +1,804 @@
+---
+permalink: python/facet-trellis/
```
Comment (cldougl) [inline _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html]: I'll just check on the url before we merge this

File: _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html [hunk 5]
```diff
@@ -0,0 +1,804 @@
...
+name: Facet and Trellis Plots | plotly
+has_thumbnail: true
+thumbnail: thumbnail/facet-trellis-thumbnail.jpg
+layout: user-guide
+name: Facet and Trellis Plots
+language: python
+title: Python Facet and Trellis Plots | plotly
+display_as: statistical
+has_thumbnail: true
+page_type: u-guide
```
Comment (cldougl) [inline _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html]: there's some extra fields here- you can simplify this by making the publisher example the same as in the readme (https://github.com/plotly/documentation/tree/source-design-merge/_posts/python) // pasted below

```
! pip install git+https://github.com/plotly/publisher.git --upgrade
import publisher
publisher.publish(
    'your-tutorial-chart.ipynb', 'python/your-tutorial-chart/', 'Your Tutorial Chart',
    'How to make your-tutorial-chart plots in Python with Plotly.',
    title = 'Python Your Tutorial Chart | Plotly',
    has_thumbnail='true', thumbnail='thumbnail/your-tutorial-chart.jpg', 
    language='python', 
    # page_type='example_index', // note this is only if you want the tutorial to appear on the main page: plot.ly/python
    display_as='chart_type', order=2,
    ipynb= '~notebook_demo/1')
```

File: _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html [hunk 6]
```diff
@@ -0,0 +1,804 @@
+---
+permalink: python/facet-trellis/
+description: How to make Facet and Trellis Plots in Python with Plotly.
+name: Facet and Trellis Plots | plotly
+has_thumbnail: true
+thumbnail: thumbnail/facet-trellis-thumbnail.jpg
```
Comment (cldougl) [inline _posts/python/statistical/facet-trellis/2015-06-30-facet-and-trellis-plots.html]: nice thumbnail!

**Comment by cldougl** [comment]
@Kully this lgtm -- can you just update the version check cell to reflect the new plotly.py version (cause that's where hist traces are available right?)
then good to 💃

**Pull request merged**